### PR TITLE
fix attributeset_get_first(): set is NULL (gtk3 terminal widget)

### LINKED ISSUE #125: attributeset_get_first(): set is NULL (gtk3 terminal widget)
Reported in https://forum.puppylinux.com/viewtopic.php?p=39717#p39717
> Gtkdialog [...] GTK3 and libvte support, so UExtract is using gtkdialog's `<terminal>` widget [...]
> However, `<terminal>` widget in Vanilla [...] does nothing on exit.

This error message is output along others: "attributeset_get_first(): set is NULL"

Reproduce by (needs GTK3 with libvte2.91):

```sh
MAIN_DIALOG='
<window>
	<vbox>
		<terminal argv0="/bin/sh"
			has-focus="true">
			<variable>vte0</variable>
	    	<input>echo "echo wait for it...; sleep 3 && exit"</input>
			<action signal="child-exited">exit:EXIT</action>
		</terminal>
	</vbox>
</window>' gtkdialog -c
```
Expected output: the vte terminal should exit and print EXIT="EXIT"
Instead it does not exit.

### PR DESCRIPTION
**Pull request by step-**
Fix #125. This only fixes the "no exit" bug, not the VTE_CRITICAL color background error reported in thread https://forum.puppylinux.com/viewtopic.php?p=39717#p39717.

### CODE REVIEW

File: src/signals.c [hunk 1]
```diff
@@ -633,8 +633,18 @@ void on_any_widget_icon_release_event(GtkWidget *widget,
  *                                                                     *
  ***********************************************************************/
 
+#if HAVE_VTE
+#if VTE_CHECK_VERSION(0,36,1)
```
Comment (dimkr) [inline src/signals.c:637]: Is this the right check?

https://developer-old.gnome.org/vte/0.36/VteTerminal.html

```
void
user_function (VteTerminal *vteterminal,
               gpointer     user_data)
```

https://developer-old.gnome.org/vte/0.38/VteTerminal.html

```
void
user_function (VteTerminal *vteterminal,
               gint         status,
               gpointer     user_data)
```

Comment (dimkr) [reply src/signals.c:637]:
```
$ git tag --contains 08c778bf4a23abe614bad294027b4e88f6b727f0 | sort -n | head -n 5
0.37.0
```

Is this the right commit? https://gitlab.gnome.org/GNOME/vte/-/commit/08c778bf4a23abe614bad294027b4e88f6b727f0

Comment (step-) [reply src/signals.c:637]: Yes, this is the right commit, `08c778bf4a23abe614bad294027b4e88f6b727f0`, and you're correct, the right check is for version 0.37.0 and not 0.36.1.   Thanks for double-checking it.

**Review by dimkr — APPROVED** [review]
If the version check is correct, this looks good. I *think* I found the right commit, where this callback gained the extra parameter.

**Pull request merged**